README.md

```diff
@@ -113,148 +111,10 @@ netlify deploy # enter ./build as the deploy path
-value. A `Results` component could then simply iterate through the `results`
-from state to render search results.
-
-**src/containers**
-
-Components in this UI are separated into "Containers" and "Components". These
-can be thought of as "Logic" and "View", respectively.
-
-"Containers" are "connected" to the "context" via a "Higher Order Component"
-(HOC) `withSearch`. This HOC simply exposes all state and actions as `props`.
-A consuming Container simply accesses those actions and state, composes
-appropriate handlers and data as props and passes them to the appropriate
-Component.
-
-**src/components**
-
-Components are simply view templates. They take `props` and render
-them in markup. They have no state of their own.
-
-**src/config**
-
-Being that this is a "generic" UI that is usable with any engine via
-configuration, it should be no surprise that there is a fair amount of logic
-around that. Configuration logic is not very useful for a code reference,
-so the majority of that logic is encapsulated here to keep other code
-references clean.
-
 ## Customization
 
-It should be feasible to use this project as a starting point for your
-own implementation. Here are a few places to look to make changes:
-
-- The styles for the entire project can be found in [src/styles](src/styles).
-  Simple style tweaks can be made here, or you could replace these styles
-  entirely with your own.
-- [src/components](src/components) contains the view templates for
-  components. Structural HTML changes can be made here.
-- If you find that you have different data or behavior requirements for
-  existing components, you can customize the component Containers in
-  [src/containers](src/containers).
-- If you find you have requirements that none of the existing components
-  satisfy, you could create an entirely new component and/or container. Use the
-  [withSearch.js](src/search-lib/withSearch.js) HOC in order to access any action or state.
-- The SearchDriver can be configured directly in [App.js](src/App.js) to do things like:
-
-  - Optimize your API calls
-  - Add additional facets and customize facet behavior
-  - Disable URL State management
-
-  A full list of configuration options can be found in [SearchDriver.js](src/search-lib/SearchDriver.js)
-
-- Eject from 'configuration'. You may choose to to delete the entire [src/config](src/config) directory, which holds the configuration logic that makes this a
-  generic UI. If you're using this as your own, production application, you likely
-  won't need this.
-
-- Lastly, if you find there is a core action or state missing, you may
-  consider updating the core logic in [src/search-lib](src/search-lib).
+This project is built with [Search UI](https://github.com/elastic/search-ui), which is a React library for building search experiences. If you're interested in using this project as a base for your own, most of
+what you'll need can be found in the Search UI documentation.
```

Comment:
Lint nit - this can probably be all one line/paragraph instead of splitting it up here